src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -56,8 +56,10 @@ public class CustomAsyncResult : IAsyncResult
         public bool IsCompleted => throw new NotImplementedException();
     }
 
-    public class Helpers
+    public static class Helpers
     {
+        public static bool IsWindowsImplementation { get; } = false;//PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
Ah that's an accident when I was testing the managed implementation.
I basically discovered that the managed tests can be run on Windows which means I can debug them very easily using VS. 

By using ConditionalFact/ConditionalTheory I can run managed tests on Windows without hanging tests and spammy failures.
The idea behind these is that the difference isn't between windows/unix so doesn't really fit into TestPlatforms, but rather managed and Windows, where the managed implementation can also run on windows